docs/filters/filter.lua

```diff
@@ -0,0 +1,119 @@
+--[[
+function dump(o)
+   if type(o) == 'table' then
+      local s = '{ '
+      for k,v in pairs(o) do
+         if type(k) ~= 'number' then k = '"'..k..'"' end
+         s = s .. '['..k..'] = ' .. dump(v) .. ','
+      end
+      return s .. '} '
+   else
+      return tostring(o)
+   end
+end
+]]--
+
+function Header(el)
+  -- uppercase top-level headings for man pages
+  if FORMAT == "man" and el.level == 1 then
+    return el:walk{Str = function(el) return pandoc.Str(string.upper(el.text)) end}
+  -- add anchor links after headline for HTML
+  elseif FORMAT == "html" then
+    local span = pandoc.Span("", {class = 'bi bi-link-45deg', ["aria-hidden"] = "true"})
+    local link = pandoc.Link(span, "#" .. el.identifier, "Link to this section", {class = "icon-link"})
+    el.content[#el.content + 1] = pandoc.Space()
+    el.content[#el.content + 1] = link
+    return el
+  end
+end
+
+-- inline code is assumed to be in jq
+function Code(code)
+  code.classes[1] = "jq"
+  return code
+end
+
+-- code blocks are assumed to be in jq if no other language is given
+function CodeBlock(block)
+  if next(block.classes) == nil then
+    block.classes[1] = "jq"
+    return block
+  end
+end
+
+function Div(el)
+  local class = el.classes[1]
+  if class == "Examples" then
+    el = el:walk{CodeBlock = function(block) return example(block.text) end}
+
+    if FORMAT == "html" then
+      local summary = pandoc.Plain{
+        pandoc.RawInline("html",  '<summary class="text-body-secondary link-body-emphasis">'), pandoc.Str 'Examples',
```

Comment:
<img width="209" alt="Screenshot 2024-12-05 at 20 58 03" src="https://github.com/user-attachments/assets/e7f59139-7729-4f7d-9175-22cbd34213fe">
Using summary tag duplicates the triangle icon on Safari. Also, the font size is big.

Reply:
I tried to rectify this now.